docs/post-upgrade/post-upgrade-deprecated-sn-connector.asciidoc

```diff
@@ -0,0 +1,64 @@
+[[post-upgrade-deprecated-sn-connector]]
+== Update a deprecated ServiceNow connector
+
+{sn} ITSM and SecOps connectors created in {stack} version \<=7.15 are marked as deprecated after you upgrade to version >=7.16. Deprecated connectors have a yellow icon after their name and display a warning message when selected.
+
+[role="screenshot"]
+image::images/cases-deprecated-sn-connector.png[]
+
+IMPORTANT: Deprecated connectors will continue to function with the rules they were added to and can be assigned new rules. Deprecated connectors will also continue to function with the cases that they were assigned to, but cannot be assigned to new ones. We strongly recommend updating deprecated connectors or creating new ones to ensure you have access to connector enhancements, such as updating incidents.
+
+[float]
+[[pre-req-deprecated-sn-connector]]
+=== Prerequisites
+Before creating a new {sn} ITSM or SecOps connector, you must complete the following within your {sn} instance:
+
+* Install https://store.servicenow.com/sn_appstore_store.do#!/store/application/7148dbc91bf1f450ced060a7234bcb88[Elastic for ITSM] or https://store.servicenow.com/sn_appstore_store.do#!/store/application/2f0746801baeb01019ae54e4604bcb0f[Elastic for Security Operations (SecOps)] from the {sn} Store.
+* Create a {sn} integration user and assign it the appropriate roles. 
+* Create a Cross-Origin Resource Sharing (CORS) rule. 
+
+*Create a {sn} integration user*
+
+To ensure authenticated communication between Elastic and {sn}, create a {sn} integration user and assign it the appropriate roles.
+
+. In your {sn} instance, go to *System Security -> Users and Groups -> Users*. 
+. Click *New*. 
+. Complete the form, then right-click on the menu bar and click *Save*. 
+. Go to the *Roles* tab and click *Edit*.
+. Assign the integration user the following roles:  
+** `import_set_loader`
+** `import_transformer`
+** `personalize_choices`
+** `x_elas2_inc_int.integration_user` (for {sn} ITSM connectors only)
+** `sn_si.basic` (for {sn} SecOps connectors only)
+** `x_elas2_sir_int.integration_user` (for {sn} SecOp connectors only)
+. Click *Save*.
+
+*Create a CORS rule*
+
+A CORS rule is required for communication between the Elastic and {sn}. To create a CORS rule:
+
+. In your {sn} instance, go to *System Web Services -> REST -> CORS Rules*.
+. Click *New*.
+. Configure the rule as follows:
+* *Name*: Name the rule.
+* *REST API*: For {sn} ITSM connectors, choose `Elastic ITSM API [x_elas2_inc_int/elastic_api]`. For {sn} SecOps connectors, choose `Elastic SIR API [x_elas2_sir_int/elastic_api]`.
+* *Domain*: Enter the Kibana URL.
+. Go to the *HTTP methods* tab and select *GET*.
```

Comment:
```suggestion
. Select the *HTTP methods* tab, then click *GET*.
```

Reply:
Same comment as above, users don't need to actually click on the **HTTP methods** tab to access and select the **GET** option.